### PR TITLE
Support Multiple PR Previews

### PR DESCRIPTION
**Pull request by Shaikh-Ubaid**
towards https://github.com/lfortran/lcompilers_frontend/issues/67

### CODE REVIEW

**Comment by Shaikh-Ubaid** [comment]
It seems to work after commit [Create .nojekyll](https://github.com/lfortran/pull_request_preview/commit/33ba9fcfc6b0c3b3b8928378fc5f93bc39be0df3)

File: utils/commit.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "id": "c969e0bc3" }
```
Comment (Shaikh-Ubaid) [inline utils/commit.json:1]: With this PR, we are now maintaining the `latest_commit` in this `utils/commit.json` file. (It is now separate from the `build` script).

**Comment by Shaikh-Ubaid** [comment]
Currently, after a PR is closed/merged, its preview folder needs to be manually deleted. (Automatic deletion of the preview folder is yet to be supported.)

**Review by certik — APPROVED** [review]
I think this looks good. After you merge, can you please also document in the README what the process is to update LFortran?

**Pull request merged**